### PR TITLE
[wagdie-subgraph] add: New vote weights if staked or seared.

### PR DESCRIPTION
**Pull request by LookingForOwls**
Changes proposed in this pull request:
- Update vote calculations to give bonus to WAGDIE NFTs if staked or seared.

### CODE REVIEW

**Comment by ChaituVR** [comment]
@IcculusHerEmissary can take a look?

**Comment by IcculusHerEmissary** [comment]
@ChaituVR yes this change is correct and approved by us. Thank you for checking with me.

**Review by IcculusHerEmissary — APPROVED** [review]
- We requested this change from @LookingForOwls and it is functioning as expected.

**Comment by IcculusHerEmissary** [comment]
Anything else I need to do for this PR?

File: src/strategies/wagdie-subgraph/index.ts [hunk 1]
```diff
@@ -5,8 +5,8 @@ const SUBGRAPH_URL = {
   '1': 'https://api.thegraph.com/subgraphs/name/wagdie/wagdieworld-mainnet'
 };
 
-export const author = 'IcculusHerEmissary';
-export const version = '0.1.0';
+export const author = 'LookingForOwls';
```
Comment (ChaituVR) [inline src/strategies/wagdie-subgraph/index.ts:8]:
```suggestion
export const author = 'IcculusHerEmissary';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Everything else looks good ✅ 
No open proposals ✅

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**